packages/ag-grid-theme/css/odyssey-ag-theme.scss

```diff
@@ -453,18 +490,18 @@ $ag-theme-odyssey-params-touch-dark: map-merge(
   );
   --agGrid-checkbox-background-selected: var(
     --uitkAgGrid-checkbox-background-selected,
-    var(--uitk-selectable-background-selected)
+    var(--uitk-selectable-borderColor-selected)
   );
   --agGrid-checkbox-foreground: var(
     --uitkAgGrid-checkbox-foreground,
-    var(--uitk-text-primary-foreground)
+    var(--uitk-selectable-foreground)
   );
   --agGrid-checkbox-foreground-selected: var(
     --uitkAgGrid-checkbox-foreground-selected,
-    var(--uitk-text-primary-foreground)
+    var(--uitk-selectable-primary-foreground-selected)
   );
-  --_agGrid-checkbox-border: var(--uitk-selectable-borderWidth)
-    var(--uitk-selectable-borderStyle) var(--uitk-selectable-borderColor);
+  --_agGrid-checkbox-border: 1px var(--uitk-selectable-borderStyle)
```

Comment:
```suggestion
  --agGrid-checkbox-border: 1px var(--uitk-selectable-borderStyle)
```

Reply:
This is a "private" css variable so the underscore was intentional.